draft-tschofenig-tls-extended-key-update.md

```diff
@@ -440,6 +441,23 @@ is the concatenation of the key_exchange field for each of the algorithms.
 The same approach is then re-used in the extended key update when
 key shares are exchanged.
 
+# SSLKEYLOGFILE update
+
+As Extended Key Update invalidates previous secrets, SSLKEYLOGFILE {{I-D.ietf-tls-keylogfile}} needs to
+be populated with new entries. Each completed Extended Key Update results
+in two additional secret labels in SSLKEYLOGFILE:
+
+1. `CLIENT_TRAFFIC_SECRET_N+1`: identified as client_application_traffic_secret_N+1 in the key schedule
+
+2. `SERVER_TRAFFIC_SECRET_N+1`: identified as server_application_traffic_secret_N+1 in the key schedule
+
+Similarly to other records in SSLKEYLOGFILE label is followed by 32-byte value
+of the Random field from the ClientHello message that established the TLS
+connection and corresponding secret encoded in hexadecimal.
+
+SSLKEYLOGFILE entries for Extended Key Update MUST NOT be produced if
+SSLKEYLOGFILE was not used for other secrets in the handshake.
+
```

Comment:
We may want to add the following text to this section : 
Note that even with knowledge of '_TRAFFIC_SECRET_N+1', none of the subsequent application data traffic secrets generated using extended key update can be derived.

Reply:
I'm not sure if I fully understand this suggestion. `CLIENT_TRAFFIC_SECRET_N+1` and `SERVER_TRAFFIC_SECRET_N+1` would provide secrets to decrypt application data traffic right after the Extended Key Update.
Are you referring to the fact that new set of SSLKEYLOG entries needs to be generated for each Extended Key Update?

Reply:
Yes, I am referring that a new set of SSLKEYLOG entries need to be generated for each Extended Key Update.

Reply:
Addressed the suggestion with following wording:

Note that each successful Extended Key Update invalidates all previous SSLKEYLOGFILE secrets including
past iterations of `CLIENT_TRAFFIC_SECRET_` and `SERVER_TRAFFIC_SECRET_`.